data/korath/korath missions.txt

```diff
@@ -257,3 +257,331 @@ mission "Small Scale Delivery"
+
+			`	The crew makes a last sweep to move debris, both to assess for repairs later and to make sure you haven't missed anyone, or parts of them. You clean and disinfect as best you can and return to your ship for the trip to <destination>.`
+				to display
+					"flagship crew" > 1
+			`	As a precaution, you make a last sweep to move debris, both to assess for repairs later and to make sure you haven't missed anyone, or parts of them. You clean and disinfect as best you can and return to your ship for the trip to <destination>.`
+				to display
+					"flagship crew" <= 1
+			branch accept
+	on complete
+		conversation
+			`A crowd pours out of the spaceport when you land. It takes some time for the workers to establish barricades and clear a path so the transports can get to your cargo hold. A team in uniform greets you formally when you open the hatch, and you count the coffins together with them. Afterwards, you show them the boxes of possessions, which they handle with care. An officer thanks you and hands you a data chip that vaguely reminds you of a license.`
+			`	The crowd has been somber, but when the first coffin leaves your ship, they erupt in a cacophony of barking sounds. Some Korath try to rush forward but are held back by others, presumably their family.`
+			`	Rather than intrude on their grief, you return to your ship, slotting the data chip you received into your systems. Unfortunately, the information is in an alien language, so you set it aside and work on a plan to salvage the <EDC-ship>. Gathering some notes on the ship and the number of crew they were carrying, it's clear that you will need to carry at least 66 additional crew that you can communicate with to be able to salvage it. The closest system that comes to mind with many human speakers is on Greenwater, in Hai space.`
+	npc
+		system "Seketra"
+		fleet "Small Kor Mereti"
+		government "Kor Mereti"
+		to spawn
+			random < 30
+		personality
+			staying
+
+mission "Efreti: Distress Call 5"
+	name "<EDC-ship> salvage team"
+	description "Hire a <bunks> member salvage team on <destination> to salvage the Efreti ship <EDC-ship>."
+	source "Karek Fornati"
+	destination "Greenwater"
+	landing
+	passengers 66
+	
+	to offer
+		has "Efreti: Distress Call 4: done"
+	
+	on complete
+		conversation
+			`Many stories tell of Hai who like change and excitement, but their enthusiasm seems to have died. Thoughts of the species that invaded them still inspire fear. No Hai have the courage to salvage a ship that has been caught in the crossfire of the Mereti and Sestor war.`
+			`	Fortunately, there are plenty of Humans who are willing to risk it for the right pay, even given the comfortable life they have in Hai space. You show the new crew members to their bunks and prepare to salvage the <EDC-ship>.`
```

Comment:
This mission requires passenger space, but has no blocked message, and no `on offer` conversation.

I suggest moving the `on complete` conversation from the previous mission into the `on offer` of this one, removing the passengers from this, and move the `on complete` conversation from this mission the `on offer` of the next one.

Reply:
```suggestion
mission "Efreti: Distress Call 5"
	name "<EDC-ship> salvage team"
	description "Hire a <bunks> member salvage team on <destination> to salvage the Efreti ship <EDC-ship>."
	source "Karek Fornati"
	destination "Greenwater"
	landing
	passengers 66
	blocked `You need <bunks> available to carry the salvage team.`
	
	to offer
		has "Efreti: Distress Call 4: done"
	
	on complete
		conversation
			`Many stories tell of Hai who like change and excitement, but their enthusiasm seems to have died. Thoughts of the species that invaded them still inspire fear. No Hai have the courage to salvage a ship that has been caught in the crossfire of the Mereti and Sestor war.`
			`	Fortunately, there are plenty of humans who are willing to risk it for the right pay, even given the comfortable life they have in Hai space. You show the new crew members to their bunks and prepare to salvage the <EDC-ship>.`
```

Is this `blocked` message good? (Also, I think human should be lowercase)